.github/workflows/cd.yaml

```
@@ -143,6 +148,9 @@ jobs:
           path: deployment/terraform
           retention-days: 1
 
+      - name: 'Verify did endpoint is available'
+        run: curl --fail $DID_ENDPOINT | jq '.id'
```

nit: Should we add retry here as well ? I am not sure, so if you think it is not needed then we ignore it. thanks.
```suggestion
        run: curl --retry 6 --fail $DID_ENDPOINT | jq '.id'
```